### ISSUE TITLE
[Accessibility] DescriptionFormatter does not work with heatmaps.

### ISSUE THREAD

**Issue by magdalena-gut** [highcharts/highcharts#16680]
#### Expected behaviour
DescriptionFormatter should allow overriding point description while using the accessibility module.

#### Actual behaviour
Using DescriptionFormatter does not override description as the code would suggest, but the default valueDescriptionFormat disappears.

Demo:
https://jsfiddle.net/BlackLabel/rmk9fw02/

#### Affected browser(s)
N/A

**Comment by raf18seb** [comment]
Hi @magdalena-gut, thanks for creating the ticket!

I used the NVDA screenreader, newest Chrome 96, Windows 10 - the `point.descriptionFormatter` is being read correctly. Could you provide the exact steps to reproduce the issue?

**Comment by raf18seb** [comment]
After further investigation:
The issue is about saving the description value on the points' object `chart.series[0].points[0].accessibility`).
With default description, the value is saved on point here:https://github.com/highcharts/highcharts/blob/789a317bde7548194c98c35f325246dbfab24306/ts/Accessibility/Components/SeriesComponent/SeriesDescriber.ts#L509
![image](https://user-images.githubusercontent.com/23310742/142645353-167152e2-8d4e-4689-b6df-350333c3b681.png)


But the formatter's value is not saved anywhere on a point:


![image](https://user-images.githubusercontent.com/23310742/142645505-5c44c52c-a53f-4979-9199-1e9e93f449f7.png)

It's not really a bug, it's just a lack of information stored on a point.
However, the information can be found in `chart.series[0].points[0].graphic.element.ariaLabel`

Closing as not a bug and resolved.

**Comment by magdalena-gut** [comment]
Thank's @raf18seb for clarifying this!
